templates/pipelines/pipeline_builder.html

```diff
@@ -18,6 +18,14 @@
       top: 50%;
       transform: translate(0, -50%);
     }
+
+    .handle-text {
+      font-weight: bold;
+      position: relative;
+      width: 100px;
+      transform: translate(15px, -15px);
+      pointer-events: none;
+    }
```

Comment:
This could be replaced with the following tailwind classes:

```
font-bold relative w-24 translate-x-[15px] -translate-y-[15px] pointer-events-none
```